builtin/game/register.lua

```diff
@@ -118,6 +118,12 @@ function core.register_item(name, itemdef)
 	end
 	itemdef.name = name
 
+	-- default description to item name
+	itemdef.description = itemdef.description or name
+	-- default short_description to first line of description
+	itemdef.short_description = itemdef.short_description or
+		itemdef.description:split("\n", false, 1, false)[1]
```

Comment:
gsub should be faster here than constructing a temporary table

Reply:
I'm not experienced with `gsub`. Would the following be correct:
`itemdef.description:gsub("^([^\n]*)\n(.*)", "%1")`

Reply:
`itemdef.description:gsub("\n.*","")` works just as well

Reply:
Thanks, done.